doc/source/changelog.rst

```diff
@@ -1,13 +1,17 @@
 MontePy Changelog
 =================
 
-#Next Version#
---------------
-
+0.4.0
+-----
```

Comment:
Not the time or place to change this.

Reply:
Should I separate them into "#Next Minor Version#" and "#Next Patch#"?

Reply:
No the plan is to do one big release soon

Reply:
#Next Version# is used by `grep`